packages/core/src/writers/schemas.ts

```diff
@@ -148,18 +171,30 @@ export const writeSchemas = async ({
         ? fileExtension.slice(0, -3)
         : fileExtension;
 
+      let namingConventionTransform = camel;
```

Comment:
Let's refactor the process. Add a function like `conventionName(name, convention)` below:

https://github.com/orval-labs/orval/tree/master/packages/core/src/utils/case.ts

Reply:
Ok. Done 👍🏽